Processes/Docs-Contribution-Process-External.md

```diff
@@ -0,0 +1,44 @@
+# Docs Contribution Process - External
+
+## Overview
+This document provides instructions for contributors external to the ENF Foundation on how to contribute new or updated content to [EOS Documentation](https://docs.eosnetwork.com).
+
+### When to Use this Procedure
+Use this procedure when you need to update content from one or repositories to  [EOS Documentation](https://docs.eosnetwork.com).
+
+### Expected Results
+Fresh, updated content is available for consumption in EOS Documentation.
+
+## How to Contribute Content to EOS 
+1. Fork the docs repository
+2. Create a new branch
+3. Create/edit/rename files
+4. Create a Pull Request with the target as this repository's main branch
+5. Request a review
```

Comment:
The act of creating a pull request is already requesting a review, we can remove step 5

Reply:
Done